README.md

```diff
@@ -1,25 +1,55 @@
-# V for Visual Studio Code
+# V Language support for Visual Studio Code
```

Comment:
`Language` should be lowercase:

~~~markdown
# V language support for Visual Studio Code
~~~